vue-components/tests/unit/components/Dropdown.spec.ts

```diff
@@ -5,9 +5,11 @@ import OptionsMenu from '@/components/OptionsMenu.vue';
 import ValidationMessage from '@/components/ValidationMessage.vue';
 import { MenuItem } from '@/components/MenuItem';
 
-async function createDropdownWrapperWithExpandedMenu( menuItems: MenuItem[] ): Promise<Wrapper<Dropdown>> {
+async function createDropdownWrapperWithExpandedMenu( menuItems: MenuItem[],
+	value?: Record<string, any> | null ): Promise<Wrapper<Dropdown>> {
 	const wrapper = mount( Dropdown, { propsData: {
 		menuItems,
+		value: value || null,
```

Comment:
This can be simplified a tiny bit:
```suggestion
	value?: MenuItem ): Promise<Wrapper<Dropdown>> {
	const wrapper = mount( Dropdown, { propsData: {
		menuItems,
		value,
```

We don't need to explicitly pass `null` because the component prop will default to `null` on its own if it's undefined. We can also use `MenuItem` instead of a `Record` because that's exactly what the Dropdown expects.

Reply:
Makes sense! Committing. Thanks, Jakob 💯